src/types.ts

```diff
@@ -222,14 +296,37 @@ type TransactionAddress = {
   type: TransactionAddressType
 }
 
+type TransactionSumEvidence = {
+  currency?: string
```

Comment:
Ok! I was looking at `PaymentValue` - but it makes sense that they should be non-nullable since PaymentValue is used on the request, and not the response.

```
type PaymentValue = {
  amount: number
  currency?: string
}
```